content/about/how-it-works.md

```diff
@@ -4,15 +4,15 @@ alwaysopen: true
 weight: 1
 ---
 
-Though Workers Ecosystem behaves similar to JavaScript in the browser or in Node.JS, there are a few subtle differences in how you have to think about your code. Under the hood, the Workers Runtime uses the [V8 engine](https://v8.dev/) - the same engine used by Chromium and Node.JS. The Workers runtime also implements many of the standard [APIs](/reference/runtime/apis) available in most modern browsers.
+Though Cloudflare Workers behaves similar to JavaScript in the browser or in Node.JS, there are a few subtle differences in how you have to think about your code. Under the hood, the Workers runtime uses the [V8 engine](https://v8.dev/) - the same engine used by Chromium and Node.JS. The Workers runtime also implements many of the standard [APIs](/reference/runtime/apis) available in most modern browsers.
```

Comment:
"behaves" --> "behave"

Reply:
Node.JS -> Node.js

Reply:
Node.js fixed

Reply:
There are two instances of this, only one is fixed

Reply:
I fixed the other ones     👍